### ISSUE TITLE
Add a link to `docs.starport.network` from `/starport`

### ISSUE THREAD

**Issue by fadeev** [cosmos/cosmos.network#143]
We've published [documentation for Starport](https://docs.starport.network), it would be great to link to it from the Starport page.

**Comment by barriebyron** [comment]
Hi @lovincyrus, I’m up for adding links to the Starport docs https://docs.starport.network/  for this issue

I found this file https://github.com/cosmos/cosmos.network/blob/develop/pages/starport.vue and now… I realize I don’t know the syntax to add an external URL link

**Comment by lovincyrus** [comment]
There is no one particular cta that points to the starport docs previously. We need to add a button to somewhere on `/starport` cc: @miguel-tendermint

**Comment by lovincyrus** [comment]
Added the docs cta here, wdyt? @miguel-tendermint 

![image](https://user-images.githubusercontent.com/1021101/115766843-492e7700-a35d-11eb-939c-d48d92941c38.png)

**Comment by miguel-tendermint** [comment]
@lovincyrus I would suggest that the CLI should be the main call to action and have the docs as the secondary one! Does that make sense for everyone?

**Comment by barriebyron** [comment]
Hi @miguel-tendermint Oh, I see what you did. You added a docs button (cool).
@fadeev what do you think about the Launch CLI button (opens gitpod).

**Comment by fadeev** [comment]
I agree with @miguel-tendermint: primary CTA to launch the CLI and secondary button to open docs.
